static/templates/project/authoring.html

```diff
@@ -99,17 +99,26 @@
                         </md-button>
                     </div>
                 </div>
-                <div class="_authoring_content" layout="row" ng-show="project.isExpanded">
-                    <div class="md-datepicker-input _mr-24" layout="column">
-                        <label>Deadline</label>
-                        <md-datepicker class="md-input-has-value larger" name="deadline"
-                                       ng-model="project.project.deadline" md-min-date="project.minDate" md-placeholder="Deadline">
-                        </md-datepicker>
+                <div class="_authoring_content" ng-show="project.isExpanded">
+                    <div layout="row">
+                        <div class="md-datepicker-input _mr-24" layout="column">
+                            <label>Deadline</label>
+                            <md-datepicker class="md-input-has-value larger" name="deadline"
+                                           ng-model="project.project.deadline" md-min-date="project.minDate"
+                                           md-placeholder="mm/dd/yyyy">
+                            </md-datepicker>
+                        </div>
+                        <md-input-container class="md-block larger">
+                            <label>Time Allotted (in min.)</label>
+                            <input name="timeout" type="number" ng-model="project.project.timeout" min="1">
+                        </md-input-container>
                     </div>
-                    <md-input-container class="md-block larger">
-                        <label>Time Allotted (in min.)</label>
-                        <input name="timeout" type="number" ng-model="project.project.timeout" min="1">
-                    </md-input-container>
+                    <div>
+                        <div class="_sub-header">Post my task to Mechanical Turk if nobody on Daemo is available</div>
+                        <md-switch ng-change="project.AWSChanged($event)" ng-model="project.project.post_mturk">Post to Mechanical Turk</md-switch>
+                        <div class="_small-note _error">{{ project.AWSError }}</div>
```

Comment:
I think better to use this image to avoid confusion on which MTurk we are referring to.
https://images-na.ssl-images-amazon.com/images/G/01/webservices/mechanical-turk/logoAI3.gif